src/sensors/AdafruitNXPsensor.cpp

```diff
@@ -0,0 +1,291 @@
+/*
+    SlimeVR Code is placed under the MIT license
+    Copyright (c) 2021 Eiren Rain & SlimeVR contributors
```

Comment:
Copyright invalid, should be 2022 \<your name\>

Reply:
Updated